### ISSUE TITLE
charts not working in chrome 

### ISSUE THREAD

**Issue by smustaoglu** [start-angular/sb-admin-angular#14]
charts not working in chrome.Line Chart panel contains empty body.
it is working in ie 11.

**Comment by smustaoglu** [comment]
it runs after refresh the page.

**Comment by Amith92** [comment]
Hi @smustaoglu , pull the latest and let me know if you are facing the same issue. There was an issue with oclazyload.

**Comment by smustaoglu** [comment]
thanks for update. it works great.

```
 On Thursday, July 2, 2015 10:12 AM, Amith <notifications@github.com> wrote:
```

 Closed #14.—
